Makefile

```diff
@@ -0,0 +1,48 @@
+# based on templates:
+# * https://gist.github.com/turtlemonvh/38bd3d73e61769767c35931d8c70ccb4
+# * https://vincent.bernat.im/fr/blog/2017-makefile-pour-golang
+
+BINARY = gopher-ball
+GOARCH = amd64
+
+VERSION?=?
+COMMIT=$(shell git rev-parse HEAD)
+BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
+
+CURRENT_DIR=$(shell pwd)
+BUILD_DIRNAME = dist
+BUILD_DIR=${CURRENT_DIR}/${BUILD_DIRNAME}
+ASSETS_DIRNAME=assets
+
+# Setup the -ldflags option for go build here, interpolate the variable values
+LDFLAGS = -ldflags "-s -X main.VERSION=${VERSION} -X main.COMMIT=${COMMIT} -X main.BRANCH=${BRANCH}"
+
+# Build the project
+all: clean prepare linux darwin windows
+
+prepare:
+	mkdir -p dist
+
+linux:
+	echo "Skipping Linux ..."
+
+windows:
+	echo "Skipping Windows ..."
+
+darwin:
+	CGO_ENABLED=1 GOOS=darwin GOARCH=${GOARCH} go build ${LDFLAGS} -o ${BINARY}-darwin-${GOARCH}.app .
```

Comment:
with `CGO_ENABLED=0` (which seems to be the flag to set to have a static build), I'm getting the following error:

```
# github.com/veandco/go-sdl2/sdl
../../veandco/go-sdl2/sdl/yuv_sw_c.go:17: undefined: Surface
../../veandco/go-sdl2/sdl/yuv_sw_c.go:18: undefined: Surface
make: *** [darwin] Error 2
```